### ISSUE TITLE
Rename to "FSharp.Azure.CloudAgent"?

### ISSUE THREAD

**Issue by dsyme** [fsprojects/FSharp.CloudAgent#9]
@isaacabraham  - the usual expected issue from me :)  

Could we consider renaming this to FSharp.Azure.CloudAgent?  It seems the logical and consistent name for the framework?

**Comment by isaacabraham** [comment]
Yes, we could - unless we were to consider Azure as a provider that the Agent framework sat on top of? e.g. Amazon or other service bus style implementations? It's unlikely I suppose.
